src/screens/GurbaniNavigator.tsx

```diff
@@ -45,24 +37,40 @@ const getOptions = ( {
         iconName="menu"
         testID="navbar-menu"
         disabled
+        style={styles.left}
       />
     </HeaderButtons>
   ),
   headerRight: () => (
     <HeaderButtons HeaderButtonComponent={IconHeaderButton}>
+      {xlarge
+      && (
```

Comment:
```suggestion
      {xlarge && (
```

[style] getting this on the same line will make it consistent with the closing braces!